src/test/java/com/google/sps/AttendeeTest.java

```diff
@@ -0,0 +1,52 @@
+import java.util.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.text.ParseException;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import org.junit.After;
+import org.junit.Before;
+import com.google.sps.data.Attendee;
+import com.google.sps.data.AttendeeInterface;
+
+@RunWith(JUnit4.class)
+
+public class AttendeeTest {
+  // Creates test data. 
+  Date date = new Date();
+  AttendeeInterface attendee = new Attendee("12345", "Taniece", date);
```

Comment:
Dont do this. Create the test data in the test cases. Only make things instance variables if they *need* to be. Dont do it purely out of convenience.